### PR TITLE
Story/971/5071 Fix tabulator NPE on unresolved references

### PR DESCRIPTION
**Pull request by hugohills-regnosys**
Please include a summary of the change and the issue/story number.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

### CODE REVIEW

File: rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/object/ConfigurableTypeTabulatorTest.xtend [hunk 1]
```diff
@@ -1,15 +1,71 @@
...
+
+import static org.hamcrest.MatcherAssert.*
+import static org.junit.jupiter.api.Assertions.*
+
+import static extension com.regnosys.rosetta.tests.util.CustomConfigTestHelper.*
 
 class ConfigurableTypeTabulatorTest {
+
+	@Test
+	def void shouldGenerateTabulatorsForSingleTypeListedInConfig() {
```
Comment (ja6a-regnosys) [inline rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/object/ConfigurableTypeTabulatorTest.xtend:16]: what is the significance of this test?

**Review by ja6a-regnosys — COMMENTED** [review]
I'll wait for a snapshot, confirm against the test harness before approving.

File: rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/object/ConfigurableTypeTabulatorTest.xtend (hunk 1 above)
Comment (hugohills-regnosys) [reply rosetta-testing/src/test/java/com/regnosys/rosetta/generator/java/object/ConfigurableTypeTabulatorTest.xtend]: Testing tabulation of the an the simplest possible type using the default testing namespace com.rosetta.test.model which was previously not working

**Review by ja6a-regnosys — APPROVED** [review]
Brilliant work

**Pull request merged**